website/docs/docs/dbt-versions/release-notes/06-July-2023/faster-run.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Enhancement: Faster run starts and unlimited job concurrency"
+description: "We have enhanced the dbt Cloud Scheduler by reducing prep time for all accounts and provided unlimited job concurrency for Enterprise accounts."
+sidebar_label: "Enhancement: Faster run starts and unlimited job concurrency"
+tags: [07-2023, scheduler]
+date: 2023-07-06
+sidebar_position: 10
+---
+
+We’ve introduced significant improvements to the dbt Cloud Scheduler, offering improved performance, durability, and scalability. 
+
+Read more on how you can experience faster run start execution and how enterprise users can now run as many jobs concurrently as they want to.
+
+## Faster run starts
+
+The Scheduler takes care of preparing each dbt Cloud job to run in your cloud data platform. This [prep](/docs/deploy/job-scheduler#scheduler-queue) involves readying a Kubernetes pod with the right version of dbt installed, setting environment variables, loading data platform credentials, and git provider authorization, amongst other environment-setting tasks. Only after the environment is set up, can dbt execution begin. We display this time to the user in dbt Cloud as “prep time”.
+
+<Lightbox src="/img/run-start.jpg" width="85%" title="The scheduler prepares a job for execution and displays it as 'prep time' in dbt Cloud."/>
+
+For all its strengths, Kubernetes has challenges, especially with pod management impacting run execution time. We’ve rebuilt our scheduler by ensuring faster job execution with a ready pool of pods to execute customers’ jobs. This means you won't experience long prep times at the top of the hour, and we’re determined to keep runs starting near instantaneously. Don’t just take our word, review the data yourself.
+
+<Lightbox src="/img/prep-start.jpg" width="85%" title="Job prep time data has seen a 75% speed improvement from Jan 2023 to July 2023. Prep time took 106 secs in Jan and now takes 27 secs as of July."/>
+
+Jobs scheduled at the top-of-the-hour used to take over 106 seconds to prepare because of the volume of runs the scheduler has to process. Now, even with increased runs, we have reduced prep time to 27 secs (at a maximum) &mdash; a 75% speed improvement for runs at peak traffic times!
```

Comment:
```suggestion
Jobs scheduled at the top of the hour used to take over 106 seconds to prepare because of the volume of runs the scheduler has to process. Now, even with increased runs, we have reduced prep time to 27 secs (at a maximum) &mdash; a 75% speed improvement for runs at peak traffic times!
```